contracts/zero-ex/scripts/rollback.ts

```diff
@@ -386,7 +386,7 @@ async function getMainnetContractAsync(): Promise<wrappers.IZeroExContract> {
     const proxyFunctions = await querySubgraphAsync();
     const deployments = reconstructDeployments(proxyFunctions);
 
-    while (true) {
+    for (;;) {
```

Comment:
is this good practice?

Reply:
That's what the linter asks for with the recommended rules.